shading/coral-trino-parser/build.gradle

```diff
@@ -1,11 +1,12 @@
 import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
 
 plugins {
+  id 'java'
   id 'com.github.johnrengelman.shadow'
+  id 'maven-publish' //https://docs.gradle.org/current/userguide/publishing_maven.html
+  id 'signing' //https://docs.gradle.org/current/userguide/signing_plugin.html
 }
 
-apply from: "$rootDir/gradle/java-publication.gradle"
-
 dependencies {
   compileOnly 'io.trino:trino-parser:355'
```

Comment:
Is this dependency supposed to stay compileOnly/provided?

Reply:
Updated the PR, I tried both `compileOnly` and `compile`, the results are the same.